pkg/controller/machinepool/azureactuator.go

```diff
@@ -100,6 +100,7 @@ func (a *AzureActuator) GenerateMachineSets(cd *hivev1.ClusterDeployment, pool *
 	)
 
 	// Use image set for machinepool azure platform instead of the default
+	// TODO: Pass the image to installazure.MachineSets when image able to be provided
```

Comment:
Depends on the timeline for the installer work vs aro I guess.

Reply:
Here's the installer card for the installer support https://issues.redhat.com/browse/CORS-1824

Reply:
Here's the current plan until I hear otherwise...I've put CORS-1824 as blocking HIVE-1696 for now, and we can wait for the installer support and update this PR when that lands.

Reply:
#1764 vendors the changes from from the installer which add `OSImage` to azure platform.